src/zeit/content/article/edit/browser/testing.py

```diff
@@ -75,6 +76,7 @@ def save(self, locator=None):
 
     def create_block(self, block, wait_for_inline=False):
         s = self.selenium
+        time.sleep(2)
```

Comment:
`time.sleep(X)` ist ein ganz schlechter Stil. Bei dir mag 2 Sekunden reichen, auf dem (langsamen) Jenkins vielleicht wieder nicht. Statt einer so hoffnungsvollen Zahl, die ggf. auch unnötig Zeit verspeist, lieber ein `s.waitFor` einbauen. In diesem Fall `s.waitForElementPresent('link=Struktur')`.